CODEOWNERS

```diff
@@ -9,6 +9,7 @@
 /topics/ingest-events-dlq.yaml                                 @getsentry/owners-snuba
 /topics/ingest-metrics-dlq.yaml                                @getsentry/owners-snuba
 /topics/ingest-generic-metrics-dlq.yaml                        @getsentry/owners-snuba
+/topics/ingest-feedback-events-dlq.yaml                        @getsentry/owners-snuba
```

Comment:
Isn't this replays team now?

Reply:
Oh I did this because of the pattern that all other dlq's are owners-snuba only. Would you recommend I change it to snuba+replay, or just replay?

Reply:
Hmm, probably just replay is fine if you guys own it